gremlin-go/pom.xml

```diff
@@ -70,22 +70,22 @@ limitations under the License.
                                     </arguments>
                                 </configuration>
                             </execution>
-                            <execution>
```

Comment:
Why is this commented out?

Reply:
I should actually delete this, since there is no longer `gremlin-go-godog-tests` server in our docker compose.